src/FormAdapter/StandardFormAdapter.php

```diff
@@ -79,6 +79,62 @@ public function toQuery($data, $formSettings)
         return $query;
     }
 
+    public function stringifyData($data, $page) : array
+    {
+        $dataString = [];
+        $formSettings = $page->settings()['form'];
+
+        if (!empty($data['q'])) {
+            $dataString['main_query'] = $data['q'];
+        }
+
+        if (!empty($data['filters']['queries'])) {
+            $filters = $this->stringifyFilters($data['filters']['queries'], $page, $data['filters']['match']);
+            if (!empty($filters)) {
+                $dataString["filters_subqueries"] = $filters;
+            }
+        }
+
+        foreach ($formSettings['elements'] ?? [] as $formElementData) {
+            $name = $formElementData['name'];
+            $apiManager = $this->getApiManager();
+            $formElement = $this->searchFormElementManager->get($name);
+            $dataString["form_element_$name"] = $formElement->stringifyData($data, $formElementData, $apiManager);
```

Comment:
Search form elements can be repeatable, meaning one `'name'` can be associated to multiple elements (you can have several elements in `$formSettings['elements']` that share the same name).
If that happens, you will overwrite data in `$dataString`.

I think `$dataString` doesn't need to be an associative array here. It might be handy for theme authors though... So not sure what should be done here. We can talk about it tomorrow ;)